assignments/prototypes.js

```diff
@@ -14,6 +14,13 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
+function GameObjects(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.dimensions = attributes.dimensions;
+  this.destroy = function() {
+    return `${this.name} was removed from the game.`;
```

Comment:
Does this line actually work? How does line 21 have access to this.name if you're not binding it within the GameObjects function?